### PR TITLE
Add missing Junos smoke tests.

### PR DESCRIPTION
**Pull request by danielmellado**
This PR adds the missing smoke tests that were not pulled when creating
the collection.

### CODE REVIEW

**Comment by trishnaguha** [comment]
recheck

File: tests/integration/targets/junos_smoke/tests/netconf/common_utils.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+# junos interface -> remove_default_spec() conditional()
+- debug:
+    msg=: "START junos_interface netconf/common_utils.yaml on connection={{ ansible_connection }}"
```
Comment (rohitthakur2590) [inline tests/integration/targets/junos_smoke/tests/netconf/common_utils.yaml:4]: - debug: msg="START junos_interface netconf/common_utils.yaml on connection={{ ansible_connection }}"

File: tests/integration/targets/junos_smoke/tasks/main.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+---
+- { include: cli.yaml, tags: ['cli'] }
```
Comment (rohitthakur2590) [inline tests/integration/targets/junos_smoke/tasks/main.yaml:2]: network_cli

**Comment by rohitthakur2590** [comment]
I have different open PR with incorporated comments.
https://github.com/ansible-collections/junipernetworks.junos/pull/58

**Pull request closed without merging**